src/components/parts/header/HeaderPart.tsx

```diff
@@ -11,23 +11,21 @@ export const HeaderPart = ({ config }: HeaderProps) => {
         return null;
     }
 
-    const { title, typo, titleTag, anchorId, justify } = config;
+    const { title, titleTag, anchorId, justify } = config;
```

Comment:
Typo-feltet kan være i bruk og bør beholdes. Burde oppdatere valgene i XP for å reflektere ny typografi, samtidig må vi nesten ha bakoverkompatibilitet med eksisterende verdier også.